x/cronos/client/cli/tx.go

```diff
@@ -124,7 +124,7 @@ func NewSubmitTokenMappingChangeProposalTxCmd() *cobra.Command {
 			fmt.Sprintf(`Submit a token mapping change proposal.
 
 Example:
-$ %s tx gov submit-proposal token-mapping-change gravity0x0000...0000 0x0000...0000 --from=<key_or_address>
+$ %s tx gov submit-legacy-proposal token-mapping-change gravity0x0000...0000 0x0000...0000 --from=<key_or_address>
```

Comment:
should this migrate to the new proposal command? or add it a separate one?

Reply:
it's not a trivial migration though, there are still some proposals in SDK still using the "legacy" mode, we can try that in later PR.

Reply:
ok, maybe just a new command?

Reply:
The new one will reuse the msg router, it seems any kind of msg can be executed from gov proposal, so just need to implement it as a normal msg, just deny the access from normal tx but can be executed from gov proposal.

Reply:
https://github.com/crypto-org-chain/cronos/issues/633